src/Generic/hooks/transfer-server.ts

```diff
@@ -58,11 +58,16 @@ export function useTransferInfos(
     domain => {
       return (transferInfosCache.get(domain) ||
         transferInfosCache.suspend(domain, async () => {
-          const transferServer = await initTransferServer(domain, testnet)
-          if (!transferServer) return []
+          try {
+            const transferServer = await initTransferServer(domain, testnet)
+            if (!transferServer) return []
 
-          const transferInfo = await fetchTransferInfos(transferServer)
-          return (transferInfo ? [transferInfo] : []) as [TransferServerInfo] | []
+            const transferInfo = await fetchTransferInfos(transferServer)
+            return (transferInfo ? [transferInfo] : []) as [TransferServerInfo] | []
+          } catch (error) {
+            // catch network error if no transfer server is available for domain
```

Comment:
Please log the error using `console.error()`, so that we know at least that an error occured 😉